src/ObjectStorage.php

```diff
@@ -142,11 +138,11 @@ public function offsetGet($object)
     {
         $hash = spl_object_hash($object);
 
-        if (isset($this->objects[$hash])) {
-            return $this->data[$hash];
+        if (! isset($this->objects[$hash])) {
+            throw new \UnexpectedValueException('Object not found.');
         }
 
-        throw new \UnexpectedValueException('Object not found.');
+        return $this->data[$hash];
```

Comment:
This change brings no value.

Reply:
This approach is about `error exception first`.

It's about change the order of conditions.

Reply:
I got that, but what's the point? Please don't spent time on nitpick like this :)

Reply:
Ok. I will not spend this change.